Project.toml

```diff
@@ -14,7 +13,6 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 
 [compat]
-FastGaussQuadrature = "0.4"
```

Comment:
Interesting, is there an issue loading this? This compat entry should allow the most current version. (This package provides a more performant solution to find the nodes and weights over the implemention in SpecialPolynomials.)